FocusOn/ViewModel/TodayViewModel.swift

```diff
@@ -45,10 +45,9 @@ class TodayViewModel: ObservableObject {
         try dataService.insertGoal(goal: todayGoal)
     }
     
-    func updateGoal(goal: Goal, name: String, isCompleted: Bool = false) throws {
```

Comment:
Eventually, `isCompleted` could still have a usage here. Setting it to yes would conveniently mark all its tasks as completed.

Ideally, the part of marking all tasks as completed when a goal is marked as complete should not be here but in the goal model. It's what we call a "business rule" and it's generally considered has a good practice to have business rules in the model/entity.